openlibrary/core/imports.py

```diff
@@ -203,24 +221,57 @@ def get_count_by_date_status(self, ndays=10):
         d = defaultdict(dict)
         for row in result:
             d[row.date][row.status] = row.count
-        return sorted(d.items(), reverse=True)
+        date_counts = sorted(d.items(), reverse=True)
+        return date_counts
+
+    @classmethod
+    def get_count_by_date_status(cls, ndays=10, use_cache=False):
+        if use_cache:
+            date_counts = cache.memcache_memoize(
+                cls._get_count_by_date_status,
+                "imports.get_count_by_date_status",
+                timeout=60 * 60,
+            )(ndays=ndays)
+            # Don't cache today
+            date_counts[0] = cache.memcache_memoize(
+                cls._get_count_by_date_status,
+                "imports.get_count_by_date_status_today",
+                timeout=60 * 3,
+            )(ndays=1)
```

Comment:
```suggestion
            )(ndays=1)[0]
```
Same issue as before.